pkg/operator/controller/crl/controller.go

```diff
@@ -188,7 +188,7 @@ func (r *reconciler) ingressControllersWithCRLConfigmap(name string) ([]operator
 // crlConfigmapToIngressController maps a configmap to a slice of reconcile
 // requests, one request per ingresscontroller that references the client CA
 // configmap.
-func (r *reconciler) crlConfigmapToIngressController(o client.Object) []reconcile.Request {
+func (r *reconciler) crlConfigmapToIngressController(ctx context.Context, o client.Object) []reconcile.Request {
```

Comment:
`ingressControllersWithCRLConfigmap` function down the code is similar to `ingressControllersWithConfigMap` in the way that it can get the real context now too.

Reply:
Will do.

Reply:
Done for real this time.